cmd/livekit-cli/main.go

```diff
@@ -52,9 +50,19 @@ func main() {
 				},
 			},
 		},
-	}
+		Before: func(c *cli.Context) error {
+			logConfig := &logger.Config{
+				Level: "info",
+			}
+			if c.Bool("verbose") {
+				logConfig.Level = "debug"
+			}
+			logger.InitFromConfig(logConfig, "livekit-cli")
+			lksdk.SetLogger(logger.GetLogger())
```

Comment:
Maybe switch to v2 is a good moment to use `slog`? It seems like https://github.com/uber-go/zap/issues/1333 is not there yet, but other options like https://github.com/samber/slog-zap already exist.

Reply:
we can wait till things stabilizes a bit before updating to it